backend/core/src/aggregator.rs

```diff
@@ -256,8 +257,9 @@ impl Handler<RenameChain> for Aggregator {
             }
 
             // Update UI
-            self.serializer.push(feed::RemovedChain(&entry.label));
-            self.serializer.push(feed::AddedChain(&new, entry.nodes));
+            self.serializer.push(feed::RemovedChain(entry.genesis_hash));
+            self.serializer
+                .push(feed::AddedChain(&new, entry.genesis_hash, entry.nodes));
```

Comment:
I think we can just remove these lines now, since the genesis hash is identical?

Reply:
The hash doesn't change, but the name can still change :).